src/Xamarin.Android.Tools.Bytecode/Xamarin.Android.Tools.Bytecode.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;$(DotNetTargetFramework)</TargetFrameworks>
```

Comment:
This "confuses" me, and I'm not sure what's going on.  `$(DotNetTargetFramework)` is `net7.0`, thus I would *expect* `bin/Debug-net7.0/Xamarin.Android.Tools.Bytecode.dll` to be a file that exists after a build.

However, it doesn't exist; instead, I have `bin/Debug-net6.0/Xamarin.Android.Tools.Bytecode.dll`, which is a copy of the .NET Standard 2.0 build, which is *weird*.

Meaning, as a consequence of this change, e.g. the net6.0 `class-parse.dll` now references the .NET Standard 2.0 build of `Xamarin.Android.Tools.Bytecode.dll` (v0.2.0.11), whereas before this change it referenced the net6.0 build, 7.0.0.7.

In *this* case that likely doesn't matter, as there's no `#if NET` in `Xamarin.Android.Tools.Bytecode.dll`, but if that ever changed…

I suspect what we *actually* want/need is to instead add a *3rd* `$(TargetFramework)` value in many of these places, instead of replacing `net6.0` with `net7.0`, a'la:

```
<TargetFrameworks>netstandard2.0;$(DotNetStableTargetFramework);$(DotNetTargetFramework)</TargetFrameworks
```

Reply:
Or this is a good reason to make everything target `net7.0` instead of some things still targeting `net6.0`.

Reply:
I've moved everything back to target net7.0.